app/templates/README.mdown

```diff
@@ -24,6 +24,38 @@ Edit `gulp.config.js` to set your desired ports
 
     gulp serve-dev # this will watch the .less file for changes, compile them to .css, and run the node server
 
+# Installing as service
+  
+The code includes a service script, and a service config to make installing express server service as easy as possible. The following files are involved:
+  
+- etc/init.d/node-express-service (generic express server service script)
+- etc/{app}/conf.sh (application specific service configuration, any application name allowed)
+- boot.js (entry point for express service, calls out to server.js)
+- node-app.js (required by boot.js)
+  
+The conf.sh is 'sourced' by the service script, and allows overriding the built-in defaults. Usually you only need to override SOURCE\_DIR, APP\_PORT, and ML\_PORT. Make sure they match the appropriate environment.
+
+Next install [forever](https://www.npmjs.com/package/forever) globally if it is not already installed.
+
+- `$ [sudo] npm install forever -g`
+  
+Next, push all source files to the appropriate server. The following assumes it was dropped under /space/projects/ in a folder called slush-app.live. Take these steps to install the services:
+  
+- cd /space/projects/slush-app.live
+- gulp build # this will create the ./dist/ folder with all the required assests and code
+- cd /etc
+- sudo ln -s /space/projects/slush-app.live/etc/{env} slush-app
+- cd /etc/init.d
+- sudo ln -s /space/projects/dmlc-finserv.live/etc/init.d/node-express-service slush-app
```

Comment:
You are correct.  It should.  I will fix it. I blame this on clipboard inheritance.